### PR TITLE
[7.30.x]Add Operator image tag property for overriding image streams for CVP job

### PR DESCRIPTION
**Pull request by jakubschwan**

### CODE REVIEW

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/images/Image.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
     CONTROLLER(OpenShiftConstants.KIE_IMAGE_TAG_CONTROLLER),
     KIE_SERVER(OpenShiftConstants.KIE_IMAGE_TAG_KIE_SERVER),
     MYSQL(OpenShiftConstants.KIE_IMAGE_TAG_MYSQL),
+    OPERATOR(OpenShiftConstants.KIE_IMAGE_TAG_OPERATOR),
```
Comment (sutaakar) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/images/Image.java]: Where is this enum used?

Comment (jakubschwan) [reply framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/images/Image.java]: In ImageStreamProvider class. E.g. [here](https://github.com/kiegroup/kie-cloud-tests/pull/579/files#diff-d14038884f3545bd9904c7decf0b83bbR49-R51)

Comment (jakubschwan) [reply framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/images/Image.java]: In ImageStreamProvider class. E.g. [here](https://github.com/kiegroup/kie-cloud-tests/pull/579/files#diff-d14038884f3545bd9904c7decf0b83bbR49-R51)

Comment (sutaakar) [reply framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/images/Image.java]: ok, thanks

**Pull request merged**